src/validation/stillInvolved.validation.ts

```diff
@@ -68,8 +68,8 @@ export const checkTrustLegalEntityBeneficialOwnerStillInvolved = (appData: Appli
 
 export const checkTrustIndividualBeneficialOwnerStillInvolved = (appData: ApplicationData, req: Request): ValidationError[] => {
   const allowedUrls = [
-    [config.TRUST_ENTRY_URL, config.TRUST_INDIVIDUAL_BENEFICIAL_OWNER_URL],
-    [config.TRUST_ENTRY_WITH_PARAMS_URL, config.TRUST_INDIVIDUAL_BENEFICIAL_OWNER_URL],
+    [config.REGISTER_AN_OVERSEAS_ENTITY_URL, config.TRUSTS_URL, config.TRUST_INDIVIDUAL_BENEFICIAL_OWNER_URL],
```

Comment:
is there a way of adding a unit test to cover the bug

Reply:
I will look at creating a test that sends a request with a url containing transactionId and submissionId

Reply:
Test added
Each test covers at least one validator